Assignment9/Assignment9/MainWindowViewModel.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace Assignment9
+{
+    public class MainWindowViewModel : INotifyPropertyChanged
+    {
+        public ICommand EditCommand { get; }
+        public ICommand NewContactCommand { get; }
+        public ICommand DeleteContactCommand { get; }
+        public event PropertyChangedEventHandler PropertyChanged;
+        public ObservableCollection<Contact> Contacts { get; } = new ();
+
+        private bool _edit = false; 
```

Comment:
Don't need to initialize to default value.
```suggestion
        private bool _edit; 
```